lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -99,11 +99,12 @@
         }
     }
 
-    var forEach = function (obj, length, func) {
-        var current = 0;
-
+    var forEachIfPresent = function (obj, length, func) {
```

Comment:
curious, why not use Array.prototype.forEach ?

Reply:
This was the pattern I inherited. Might be a slight difference between `Array.prototype.forEach` and the spec iteration. Might be because of surface area for tainting (would need to use `platform.ArrayInstanceForEach`). Might be a bit of both.